http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/RuntimeControlledResourceComponent.scala

```diff
@@ -38,11 +38,28 @@ object controlledResourceQuery extends TableQuery(new RuntimeControlledResourceT
       .result
       .headOption
 
+  def getWsmRecordFromResourceId(resourceId: WsmControlledResourceId,
+                                 resourceType: WsmResourceType
+  ): DBIO[Option[RuntimeControlledResourceRecord]] =
+    controlledResourceQuery
+      .filter(_.resourceId === resourceId)
+      .filter(_.resourceType === resourceType)
+      .result
+      .headOption
```

Comment:
RUNTIME_CONTROLLED_RESOURCE table is unique by runtimeId and resourceType and rather than adding a new entry when a PD is transferred to a new disk, my logic updates the entry with the new runtimeId. I.E. it should be unique by resourceId as well. I can add that constraint in the changeset as well. 

Or the alternative would be to add an entry when the PD is transferred to a new runtime, but this introduces more complications and I'd probably want to add some sort of timestamp to the table.

Reply:
That makes sense, thanks! Maybe add a small comment to explain your choice here?